test/gtfs/route_pattern_test.exs

```diff
@@ -121,4 +121,18 @@ defmodule Gtfs.RoutePatternTest do
              }
     end
   end
+
+  describe "via_variant/1" do
+    test "gets via_variant out of route_pattern_id" do
+      assert RoutePattern.via_variant("116-4-1") == "4"
```

Comment:
All of these tests use a single digit variant. Do we know that that will always be the case?

Reply:
GTFS only has single-digit variants. HASTUS has some longer variants (e.g. `5ou`), but `gtfs_creator` processes them out.